lib/apipie_bindings/api.rb

```diff
@@ -223,8 +223,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
           update_cache(response.headers[:apipie_checksum])
         rescue => e
           clear_credentials if e.is_a? RestClient::Unauthorized
-          log.debug e.message + "\n" +
-            inspect_data(e.respond_to?(:response) ? process_data(e.response) : e)
```

Comment:
> the `inspect_data` variant still can return nil, which would fail with trying to use `+` with it, so either call to_s to make sure it's a string or use string interpolation

Made it to return empty string on error.

-- 
Later,
 Lukas #lzap Zapletal